src/main/java/com/degustudios/dotnetformat/DotnetFormatRunner.java

```diff
@@ -1,25 +1,19 @@
 package com.degustudios.dotnetformat;
 
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
 import org.springframework.stereotype.Service;
+import ut.com.degustudios.bitbucket.mergechecks.NativeCommandRunner;
 
-import java.io.BufferedReader;
-import java.io.IOException;
-import java.io.InputStream;
-import java.io.InputStreamReader;
 import java.nio.file.Path;
-import java.util.concurrent.Executors;
-import java.util.function.Consumer;
 
 @Service
 public class DotnetFormatRunner {
-    private static final Logger logger = LoggerFactory.getLogger(DotnetFormatRunner.class);
+    private final NativeCommandRunner commandRunner;
 
-    public DotnetFormatRunner() {
+    public DotnetFormatRunner(NativeCommandRunner commandRunner) {
+        this.commandRunner = commandRunner;
     }
 
-    public DotnetFormatCommandResult runDotnetFormat(Path workingDirectory) {
+    public DotnetFormatCommandResult runDotnetFormat(Path workingDirectory, String param) {
         String shell;
         String executeSwitch;
         if (System.getProperty("os.name").toLowerCase().contains("win")) {
```

Comment:
Could you add test case for proper `shell` and `executeSwitch` params being selected?
(`System.setProperty` could be used in tests to define them)